modules/quality_control/README.md

```diff
@@ -0,0 +1,29 @@
+## Imaging Quality Control Module
+
+#### Purpose
+The purpose of this module is to verify and help ensure imaging data has been uploaded into LORIS and reviwed for Quality Control (QC). It provides a queryable interface for users to assess the completeness of imaging data collections matched against a given protocol, and the status of QC review processes on them.
+
+Note: this module is not the module in which visual/radiological QC is performed on scans, just the module in which QC assessments are *reviewed*. Links are included in this module so the user can jump to the Imaging Browser to mark QC assessments on images and visits.
+
+#### Intended Users
+This module is used by study administrators, imaging scientists, analysts, coordinators, and clinicians.
+
+#### Scope
+The Imaging QC module allows you to view specific data on uploaded image files, with many additional filtering options. The resulting table has a column for QC status, so you can see whether the upload has passed QC or not. The table provides a link to each file. You can download the table in `.csv` format. 
+
+Note: To QC a scan, the user must click the link leading to the Imaging Browser, and enter the QC details in that module.
+
+#### Permissions
+The `quality_control` permission gives users access to this module and its features.
+
+#### Interactions
+The Imaging Quality Control module is directly related to the following modules:
```

Comment:
I can't figure out how to do a multi-line change suggestion on GitHub, but to make this more consistent with how other README's describe interactions this should be something more like (based on how imaging browser's README is written)

```
#### Interactions With LORIS

- Links are provided to the imaging browser module and displays the QC entered there is displayed in this module[1]
- The status of excluded or failed scans from the imaging_uploader module are displayed
```

[1] I'm not sure if this is true or if I misread your comment about the link

Reply:
thanks!

Reply:
commit made -- Let us know if the content is good to go now @driusan 

@knoll-alex still needs to change the directory name for this file / address my comments, after that it should be good to merge.

Reply:
I've updated the directory name and PR but I can't seem to find the inline comments you're referring to. Can someone help populate content for the Interactions with LORIS section? I'm lost on what to include at this point.

Reply:
commit made.  i made the commit to address Dave's comments on interactions.

Reply:
re the inline comments - apologies @knoll-alex  I must have dismissed that old review, i can't see those comments now either.  
I just now re-checked it and fixed the typo, and corrected the other points I remembered, in my latest commit.  I think it's good to go now.  
Great job fixing the directory path too 👍 
I'll approve but it really needs approval by others since I've edited so much.